private/pkg/storage/mapper.go

```diff
@@ -28,12 +28,6 @@ type Mapper interface {
 	// The returned path is expected to be normalized and validated.
 	// If the path cannot be mapped, this returns false.
 	MapPath(path string) (string, bool)
-	// Map maps the prefix to the full prefix.
-	//
-	// The path is expected to be normalized and validated.
-	// The returned path is expected to be normalized and validated.
-	// If the path cannot be mapped, this returns false.
-	MapPrefix(prefix string) (string, bool)
```

Comment:
`MapPrefix` can be removed and replaced with calls to `MapPath` now that `Matcher`s won't restrict the mappers.